trogdor/README.md

```diff
@@ -6,9 +6,9 @@ Trogdor can run benchmarks and other workloads. Trogdor can also inject faults i
 
 Quickstart
 =========================================================
-First, we want to [start a single-node Kafka cluster in KRaft mode](https://github.com/apache/kafka/blob/trunk/README.md#running-a-kafka-broker-in-kraft-mode)
```

Comment:
`running-a-kafka-borker-in-kraft-mode` no longer exist by https://github.com/apache/kafka/pull/18301.
Also modified the sentence to align with the PR